pom.xml

```diff
@@ -111,11 +116,6 @@
         <artifactId>pipeline-milestone-step</artifactId>
         <version>1.3.2</version>
       </dependency>
-      <dependency>
-        <groupId>org.jenkins-ci.plugins</groupId>
-        <artifactId>ant</artifactId>
-        <version>1.13</version>
-      </dependency>
```

Comment:
now in [the bom](https://github.com/jenkinsci/bom/blob/231c8d6403cf235a1f149fd24db857395d510cad/bom-weekly/pom.xml#L151-L155)